cmd/templates-api/main.go

```diff
@@ -59,16 +61,23 @@ func main() {
 		defer tracer.Stop()
 	}
 
-	// Start server
-	if err := start(*httpHostF, *httpPortF, *repositoryPathF, *debugF, logger, envs); err != nil {
+	// Parse repositories.
+	repositories, err := parseRepositories(*repositoriesF)
+	if err != nil {
+		logger.Println(err.Error())
+		os.Exit(1)
+	}
```

Comment:
Parsovanie hodnot z `flag` som presunul sem, do `main.go`.
Je lepsie (aj lepsie testovatelne) ked sa to parsuje tam, kde je to definovane, a dalej ide uz hodnota.